_application-profiles/lotteries-io-order.md

```diff
@@ -152,7 +152,7 @@ For example:
 
 {% highlight json%}
 {
-  "order-digest": "sha256:5d5b09f6dcb2d53a5fffc60c4ac0d55fabdf556069d6631545f42aa6e3500f2e",
+  "order-digest": "SHA-256=5d5b09f6dcb2d53a5fffc60c4ac0d55fabdf556069d6631545f42aa6e3500f2e",
```

Comment:
"The canonical names are as defined in Content Signature" which lists:
md5
sha1
sha224
sha256
sha384
sha512

For which we have an utility in zoe-commons that maps those to the java digest objects.